vars/Debian.yml

```diff
@@ -0,0 +1,8 @@
+---
+setup_packages:
+  - build-essential
+python_packages:
+  - python
+  - python-pip
+  - python-setuptools
+  - python-virtualenv
```

Comment:
Per comment regarding python installation task, we need to validate this location is where we want these to live.

Reply:
Done